MapboxPlugins.podspec

```diff
@@ -30,13 +30,13 @@ s.social_media_url = "https://twitter.com/mapbox"
 
   # ――― Source Location ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.source       = { :git => "https://github.com/mapbox/mapbox-plugins-ios.git", :branch => "traffic-plugin" }
+  s.source       = { :git => "https://github.com/mapbox/mapbox-plugins-ios.git", :branch => "master" }
```

Comment:
Good catch. (We’ll need to modify this line before the next release so that CocoaPods users will get the released version instead of what’s on master.)